modules/checkout/src/Entity/CheckoutFlow.php

```diff
@@ -134,4 +134,9 @@ protected function getPluginCollection() {
     return $this->pluginCollection;
   }
 
+  public function __sleep() {
+    return parent::__sleep(); // TODO: Change the autogenerated stub
```

Comment:
This is going to be removed. But in debugging the `__sleep` method of the parent is invoking `\Drupal::service` which says that `$container` is empty.